crystal/distribution.yaml

```diff
@@ -460,6 +460,24 @@ repositories:
       url: https://github.com/eProsima/Fast-RTPS.git
       version: master
     status: developed
+  fmi_adapter_ros2:
+    doc:
+      type: git
+      url: https://github.com/boschresearch/fmi_adapter_ros2.git
+      version: 0.1.0
```

Comment:
0.1.0  appears to be a tag rather than a branch. The rosdistro checks require the `doc` and `source` version entries to be branches rather than tags.